julia-package/brainflow/Artifacts.toml

```diff
@@ -0,0 +1,7 @@
+[brainflow]
+git-tree-sha1 = "1681bc5c836c0465e02c42dce71db9572d723b6e"
+lazy = true
+
+    [[brainflow.download]]
+    sha256 = "92bd7485de8ccbd19d27a9b203e8176806269415b1087c767bb36539efe063d9"
+    url = "https://github.com/brainflow-dev/brainflow/releases/download/3.7.2/compiled_libs.tar"
```

Comment:
I will think how to update it, maybe need to check how its used in Registrator and what exactly it does

Reply:
Registrator doesnt use this file. Artifacts.toml is only used for installing and finding the binary files. 

Actually the url is now meaningless as I worked around the unpack() issue by making a manual download, but I do want to find out why we have the unpack issue and use the native artifact download functionality. We can accept the tech debt and fix it in another PR though, if you are OK?

Reply:
its ok, I was not going to change it in this pr. If its not used we probably can keep as is and dont fix it later